gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/converter/AbstractAvroToOrcConverter.java

```diff
@@ -192,6 +192,8 @@ public AbstractAvroToOrcConverter() {
     Optional<Table> destinationTableMeta = destinationMeta.getLeft();
 
     // Optional
+    Optional<String> wrapperViewName = getConversionConfig().getDestinationViewName();
+    boolean shouldUpdateView = getConversionConfig().isUpdateViewAlwaysEnabled();
```

Comment:
Why do you need another flag. If `wrapperViewName`  means the user wants to update view right?